tests/nuodb_test_class.py

```diff
@@ -34,15 +35,36 @@ def assert_await(fxn, timeout=120, interval=10):
 
     if last_assert:
         raise last_assert
+    
+def getClient():
+     influxToken = os.getenv("INFLUXDB_TOKEN", 'none')
+     org = os.getenv("INFLUXDB_ORG", "nuodb")
+     client = InfluxDBClient(url='http://localhost:8086', token=influxToken, org=org)
+     return client
+
+def getMeasurenment(client, bucket):
+    query = f"""
+    import \"influxdata/influxdb/schema\"
+
+    schema.measurements(bucket: \"{bucket}\")
+    """
```

Comment:
Is this a Python3-ism? I'm guessing the `f` prefix is what causes `bucket` to be injected into the format string without have to write `.format(bucket=bucket)`.

Reply:
Yes. F strings are sort of improved way of string formatting. It was introduced in Python 3.6